pyccl/background.py

```diff
@@ -271,35 +272,65 @@ def rho_x(cosmo, a, species, is_comoving=False):
         species_types[species], int(is_comoving))
 
 
-def Sig_MG(cosmo, a):
-    """Redshift-dependent modification to Poisson equation for massless
-    particles under modified gravity.
+def mu_MG(cosmo, a, k=None):
```

Comment:
Since this is a modification to the Poisson equation it should probably not sit under `background`, but somewhere else.

Reply:
Yeah, that was my fault, I put those there in the first place. We should move them - ideas about where?

Reply:
If a lot of this is gonna be hidden under the hood in the API, I would suggest moving them to `tracers.py` and hiding them by changing the name into e.g. `_mu_MG`.

Reply:
Fixed.  These were moved from backgound.py to musigma.py.